tools/Az.Tools.Predictor/Az.Tools.Predictor/Commands/DisableAzPredictor.cs

```diff
@@ -18,7 +18,7 @@
 namespace Microsoft.Azure.PowerShell.Tools.AzPredictor
 {
     /// <summary>
-    /// A cmdlet that disable Az Predictor.
+    /// <para type="description"> Use this cmdlet to disable Az Predictor and stop recieving suggestions</para>
```

Comment:
It requires another line for synopsis. It is required part in doc.
```
<para type="synopsis">This is the cmdlet synopsis.</para>
```

Reply:
Added the synopsis